devex/cmd/secretconverter/main.go

```diff
@@ -0,0 +1,113 @@
+package main
```

Comment:
thought (non-blocking): I forgot that I introduced this tool in the original PR :smile: